otelcollector/shared/configmap/mp/tomlparser-default-targets-metrics-keep-list.go

```diff
@@ -74,6 +74,8 @@ func populateKeepList(metricsConfigBySection map[string]map[string]string) (Rege
 				shared.SetEnvAndSourceBashrcOrPowershell("MINIMAL_INGESTION_PROFILE", minimalingestionprofile_value, true)
 			}
 		} else {
```

Comment:
Thanks Kaveesh, in the situation where if the settings configmap exists with settings but does not contain the schema version, `metricsConfigBySection` is not nil but it looks like `AZMON_AGENT_CFG_SCHEMA_VERSION`="". So would this else statement evaluate and the function return without taking the `default-targets-metrics-keep-list` values in the configmap?

Reply:
Ok thanks Kaveesh I see now that it'll always be v1, this makes sense when it's not v1 or v2.